Apt.py

```diff
@@ -0,0 +1,66 @@
+"""Apt extension
+
+Note that it is up to you to ensure that the package lists are up to date"""
+
+import os
+import subprocess
+from shutil import which
+from albertv0 import *
+import re
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Apt"
+__version__ = "1.0"
+__trigger__ = "apt"
+__author__ = "Harindu Dilshan"
+__dependencies__ = ['apt','gksu','pkexec']
+
+if which("apt") is None:
+    raise Exception("'Apt' is not in $PATH.")
+
+gksu = None
+for item in __dependencies__:
+    if which(item):
+        gksu = item
+
+UpdateActions = [TermAction("Update database in Terminal", ["sudo", "apt", "update"])]
+
+if gksu is not None:
```

Comment:
I'd recommend to remove this. You dont want to mess with the package manager. users should always see the output of it.

Reply:
which would remove gksu from the dependencies. I recommend this anyway because the workflow is exactly the same but there is one additional dependency.